keps/sig-node/34-sysctl-fields/README.md

```diff
@@ -188,15 +256,176 @@ Validation checks implemented as part of [#27180](https://github.com/kubernetes/
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, disable the feature flag.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Feature will become available again on the component.
+
+###### Are there any tests for feature enablement/disablement?
+
+Not currently. Feature has defaulted to on since 1.11; graduation criteria would lock feature to on.
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout fail? Can it impact already running workloads?
+
+N/A
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Yes: https://github.com/kubernetes/kubernetes/blob/28e2e12b887fe082929d3ece4b3cbd572dc15d39/test/e2e/upgrades/sysctl.go
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+N/A
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+No metric currently exists. Feature flag will be set to on and Pod or PSP specifications will include sysctl fields set.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+N/A, not a service.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the above SLIs?
+
+N/A, not a service.
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+N/A
+
+### Dependencies
+
+Underlying kernel support for sysctls.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No.
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+No.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Yes: pods and PSPs have new fields for sysctl values.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No.
+
+### Troubleshooting
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+Feature is an API field on pod specification; kubelets behave as usual when API server/etcd are unavailable.
+
+###### What are other known failure modes?
+
+- https://github.com/kubernetes/kubernetes/issues/72593 
+- https://github.com/kubernetes/kubernetes/issues/74151
+
+There may be some follow-ups required to improve usability, but I do not believe this should block graduation.
```

Comment:
There was a note about unsafe sysctls and usability. Maybe we can add a graduation criteria to just document that on the website with workarounds so we can graduate this rather than worry about more code fixes in this release. @pacoxu

Reply:
any scheduling enhancement we make around a node that is configured to allow unsafe sysctls would be a distinct feature.